config/install/migrate_plus.migration.y_lb_demo_page_dummy.yml

```diff
@@ -17,6 +17,8 @@ source:
     -
       id: 1
       title: 'Demo - Dummy'
+      meta_description: "Lorem ipsum dolor sit amet"
```

Comment:
Let's avoid lorem ipsum, it's better to keep it empty @aleevas @froboy

Reply:
The goal was to have the field populated, but that's fine on this page.